pkg/cnab/provider/parameters.go

```diff
@@ -83,3 +91,22 @@ func (d *Runtime) getUnconvertedValueFromRaw(def *definition.Schema, key, rawVal
 	}
 	return rawValue, nil
 }
+
+// getZeroValue returns the zero value for a parameter according to its type
+func getZeroValue(name string, def *definition.Schema, claim *claim.Claim) interface{} {
```

Comment:
looks like the claim parameter isn't used?

Reply:
Ah, thank you; indeed.